docs/layers/security-and-compliance/aws-access-analyzer.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: AWS IAM Access Analyzer
+sidebar_label: AWS Access Analyzer
+sidebar_position: 9
+description: "Identify external access to resources and unused IAM permissions"
+---
+import Intro from '@site/src/components/Intro';
+import KeyPoints from '@site/src/components/KeyPoints';
+import Note from '@site/src/components/Note';
+
+<Intro>
+AWS IAM Access Analyzer identifies resources shared with external entities and detects unused IAM permissions,
+enabling you to implement least-privilege access and identify unintended access to your resources.
+</Intro>
+
+## Overview
+
+Access Analyzer provides:
+
+- **External Access Analysis**: Identifies resources shared with external principals outside your organization
+- **Unused Access Analysis**: Detects unused IAM roles, users, and permissions
+- **Policy Validation**: Validates IAM policies against best practices
+- **Policy Generation**: Generates least-privilege policies based on CloudTrail activity
+- **Multi-account Coverage**: Organization-wide analysis from a central account
+
+## Analyzer Types
+
+This component creates two types of organization-wide analyzers:
+
+| Analyzer Type | Purpose | Findings |
+|---------------|---------|----------|
+| `ORGANIZATION` | External access analysis | Public access, cross-account access, cross-organization access |
+| `ORGANIZATION_UNUSED_ACCESS` | Unused access analysis | Unused roles, users, permissions (configurable threshold) |
+
+## Supported Resources
+
+External access analyzer monitors:
+
+- Amazon S3 buckets and access points
+- IAM roles and policies
+- AWS KMS keys
+- AWS Lambda functions and layers
+- Amazon SQS queues
+- AWS Secrets Manager secrets
+- Amazon SNS topics
+- Amazon EBS volume snapshots
+- Amazon RDS DB snapshots
+- Amazon ECR repositories
+- Amazon EFS file systems
+
+## Architecture
+
+```mermaid
+flowchart LR
+    subgraph root["Root Account"]
+        step1["STEP 1: Delegate"]
+    end
+
+    subgraph security["Security Account"]
+        step2["STEP 2: Create Analyzers"]
+        dashboard["Access Analyzer Dashboard"]
+    end
+
+    subgraph members["Member Accounts"]
+        member["Auto-analyzed"]
+    end
+
+    root -->|"Delegation"| security
+    members -->|"Findings"| dashboard
+```
+
+## Deployment
+
+Access Analyzer uses a **2-step delegated administrator** deployment model.
+
+### Step 1: Deploy to Organization Management Account
+
+<Note title="SuperAdmin Required">
+This step requires SuperAdmin permissions to deploy to the root account.
+</Note>
```

Comment:
Requires SuperAdmin specifically? We should say requires root access such as with the managers profile. We shouldnt require SuperAdmin after the cold start